<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRobotControlWidget</class>
 <widget class="QWidget" name="CRobotControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>136</width>
    <height>136</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>136</width>
    <height>136</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>136</width>
    <height>136</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="UBXPushButton" name="pbRobotLabelFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>136</width>
     <height>136</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>136</width>
     <height>136</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>136</width>
     <height>136</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbRobotHead">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>34</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>68</width>
     <height>68</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>68</width>
     <height>68</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbDisconnectRobot">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbConnectRobot">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>10</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbPowerOff">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbResetRobot">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CBatteryIcon" name="pbBattery">
   <property name="geometry">
    <rect>
     <x>43</x>
     <y>34</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="UBXPushButton" name="pbRobotWifi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>23</width>
     <height>22</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CBatteryIcon</class>
   <extends>QToolButton</extends>
   <header>../zxLib/widget/batteryicon.h</header>
  </customwidget>
  <customwidget>
   <class>UBXPushButton</class>
   <extends>QPushButton</extends>
   <header>ubxpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
